config/default/layout_builder_styles.style.no_border.yml

```diff
@@ -1,17 +1,17 @@
-uuid: 34913809-593c-48fe-adb0-4351627a7edf
+uuid: 8de239d8-731f-4623-857e-d2f255d40bf3
 langcode: en
 status: true
 dependencies: {  }
-id: card_media_position_stacked
-label: 'Stacked (default)'
-classes: card--stacked
+id: no_border
+label: 'No border'
+classes: borderless
 type: component
-group: card_media_position
+group: default
 block_restrictions:
   - 'inline_block:uiowa_card'
-  - 'inline_block:uiowa_events'
+  - 'inline_block:uiowa_event'
```

Comment:
is this supposed to be uiowa_event or uiowa_events?

Reply:
Both, I think. The style should target all blocks that get rendered as cards.

Reply:
Looks like uiowa_event doesn't have several of these style options:

<img width="694" alt="Screen Shot 2023-01-23 at 11 42 48 AM" src="https://user-images.githubusercontent.com/4663676/214111141-dca4a8be-80ac-4374-8440-f6d9465a3434.png">

Added uiowa_events to no_border. Based on the above there is more work to do on uiowa_event or it shouldn't have the no_border option.

Reply:
@bspeare Feel free to weigh in here, but generally speaking, we want all of our card-like blocks to have all the card options, right?

Reply:
The `uiowa_event` block does not have these options currently, but it will probably be nice to have those new features.  Yes, more work will need to be done for that block since nothing is rendering for it with this update.

Reply:
Updated.